drupalcr/dccr-2016
# 1: Initial scaffolding

- [x] Ensure VM builds
- [ ] Ensure aquifer doesn't exist locally
- [x] Ensure there is an easy way to build the site
- [x] Ensure site install works
- [x] Ensure behat tests work

Steps to test:
- [x] Clone
- [x] Move to right branch
- [x] Copy example.config.yml to config.yml and edit mount src path
- [x] npm install && composer install
- [x] sudo ansible-galaxy install -r provisioning/requirements.yml --force
- [x] ./scripts/local_settings.sh
- [ ] vagrant up

I'm not sure if this should build/install the site; please let me know. If not (it's still ok):
- [ ] npm run aquifer build
- [ ] ./scripts/install.sh

If you had already tried this; please destroy and recreate the VM; before that:
- [ ] npm run clean && npm install
- [ ] Remove drush aliases file (~/.drush/dccr-2016.aliases.drushrc.php)

Then:
- [ ] vagrant destroy
- [ ] Clean the build folder
- [ ] Clean the settings folder (only keep services.yml and settings.php)
- [ ] Generate settings (./scripts/local_settings.php)
- [ ] vagrant up
- [ ] Build site: npm run aquifer build
- [ ] Install site (./scripts/install.php)
